steam-market-seller.user.js

```diff
@@ -439,7 +474,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
```

Comment:
I made the same comments a year ago on https://github.com/tboothman/steam-market-seller/pull/17 when those commits were made. I don't see how it could do anything .. but he said it did. It's possible the extra asynchronicity added by all the items  being added only when the main thread is inactive makes it do something subtly different.